styles/dialog/fight.scss

```diff
@@ -23,9 +23,64 @@
         display: grid;
         width: 100%;
         height: min-content;
+        overflow: hidden;
         grid-template-columns: repeat(4, 1fr);
         margin: 1em 0 1em 0;
 
+        .pill-toggle{
+            & > input[type="checkbox"]{
+                &:checked + label {
+                    background: linear-gradient(rgb(238, 230, 126), rgb(170, 159, 0));
+                    border: 2px solid #880000;
+                    color: #880000;
+        }}}
+
+        .fighters-label{
+            width: 100%;
+            font-size: 1.25em;
+            font-weight: bold;
+        }
+
+        .portrait {
+            width: 120px; height: 120px;
+            border: 2px solid #880000;
+            object-fit: cover;
+        }
+
+        .details {
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            flex-grow: 1;
+            .reflexes {
+                padding-right: 15px;
+            }
+            td {
```

Comment:
I believe in the case of the Fight dialog, this is misuse of the rather narrow use of the `table` tag and all the related bits, so 👎 on including this in its' current state.